_data/glossary/cloud-controller-manager.yml

```diff
@@ -0,0 +1,15 @@
+id: cloud-controller-manager
+name: Cloud Controller Manager
+full-link: https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/
+related:
+- cloud controller
```

Comment:
Ok. I will add another glossary tag. Thanks for the response.

Reply:
@heckj I updated my PR. Sorry for the delay. Have been very busy with work and school.

Reply:
No problem taking the time, and no need to apologize. We appreciate the updates you're providing!

In this case, the update doesn't actually do what you want though - the value `cloud controller` is a key in YAML, so it can't support spaces in the name. While you did add a `cloud-controller`, and that could be a relevant key here, the content on that additional glossary entry is really light and hard to differentiate from this content. I think you'd be better off just removing the `related:` and `cloud controller` key and the additional glossary key, and running with the glossary content you added for `cloud-controller-manager` with this PR

Reply:
Ok. Thanks let me do that. I noticed the content on the cloud-controller was really light but I couldn't get any documentation about it online. I had to deduce it's meaning and description from the describe of cloud-controller-manager.